### DISCUSSION TITLE
An Home page with posts partially displayed

### DISCUSSION THREAD

**Discussion by jphconstantin** [mmistakes/minimal-mistakes#2568] [Q&A]
Hello,
The home page layout display mainly the name of the posts.
Is it possible to display more: e.g the first N lines of a post and a button "read more" ?
rgds,

**Comment by iBug** [comment]
The home layout should by default display post titles as well as up to 160 characters from the beginning of the post like this:

![image](https://user-images.githubusercontent.com/7273074/84666711-ff624c80-af53-11ea-91b6-8cc74dfb0cc6.png)

**Comment by mmistakes** [comment]
If you're using the the Archive layout then you should have excerpts.
https://mmistakes.github.io/minimal-mistakes/docs/layouts/#archive-layout

**Comment by jphconstantin** [comment]
The 160 characters. Before the text, I put an image but this one isn't displayed !

**Comment by n0obcoder** [comment]
how do i change the home page ?  I have "_pages/about.md" with permalink "/about/". How do I make it the homepage ?   I want it to display when I hit [username].github.io

Thanks in advance : )

**Comment by mmistakes** [comment]
@n0obcoder `permalink: /`

**Comment by n0obcoder** [comment]
@mmistakes I changed the permalink to / in "[_pages/about.md](https://github.com/n0obcoder/n0obcoder.github.io/edit/master/_pages/about.md)" 
and I also changed the URL to / in "[_data/navigation.yml](https://github.com/n0obcoder/n0obcoder.github.io/blob/6411646ee5be2d75ae0b37a6623aa4dd6f03cc1d/_data/navigation.yml#L3)"

Still it does not seem to solve the problem : /

**Comment by mmistakes** [comment]
That’s the wrong permalink. Set it to what I have above `permalink: /`

. Or rename the about.md to `index.md` and move out of _pages and into root directory.

**Comment by n0obcoder** [comment]
@mmistakes  I am sorry but i dont understand how [my](https://github.com/n0obcoder/n0obcoder.github.io/edit/master/_pages/about.md) "permalink: /" is different from the one you suggested "permalink: /"? Please don't mind me asking this again. I really can't see the difference. And thank you for giving me quick replies : )

**Comment by mmistakes** [comment]
You also have index.html in the root of your project. Delete it. It’s overriding the about page you’re trying to set as index.html

**Comment by n0obcoder** [comment]
Deleting index.html fixed the problem.  I feel embarrassed for bothering you over such an issue hahaa. I don't have much experience in web-dev :p

I appreciate you helping me out. Thanks a lot @mmistakes  :D

**Comment by jphconstantin** [comment]
What is the relation with my subject and the @n0obcoder issue ... ;-)

**Comment by n0obcoder** [comment]
@jphconstantin  there is none. I realized this a bit late, then raised a new issue. I am sorry for that : p

**Comment by jphconstantin** [comment]
You are lucky because @mmistakes is mostly rigourous ... !

**Comment by iBug** [comment]
@jphconstantin It should really be on the other side. Maintaining a large, well-received open-source project is fun, but answering all those questions, particularly when a large portion of them are raised for not reading the documentation thoroughly, or other trivial issues like not properly indenting YAML in the config file, is probably not.

**Comment by jphconstantin** [comment]
Yeah ! It was a joke !

**Comment by mmistakes** [comment]
> 
> 
> The 160 characters. Before the text, I put an image but this one isn't displayed !

@jphconstantin Please link to a public repo and where you added a teaser image via front matter so we can see what you've tried otherwise the issue will be closed.
